src/environment/Environment.cpp

```diff
@@ -28,7 +29,11 @@ void Environment::loadNavWorldInBackground() {
 }
 
 void Environment::loadConfig() {
-    config = std::make_unique<Config>(getProgramPath() + "/config.json");
+    try {
+        config = std::make_unique<Config>(getProgramPath() + "/config.json");
+    } catch (const std::exception &e) {
+        std::cerr << "Exception: " << e.what() << std::endl;

```

Comment:
I would rather add an `exit(1)` after the `std::cerr` line to exit the program at this point.
If not, then every time we call the `env->getConfig()` we must be checking if it's a nullptr before we use the config.

Reply:
Agreed. I think `config` should always be valid. My initial idea was to just add a catch handler in `Standalone.cpp` around `loadConfig`, and outputting something followed by `exit(1)` if it catches something. Then everything else would be left unaffected.